pkg/performanceprofile/cmd/render/render.go

```diff
@@ -159,6 +159,13 @@ func render(inputDir, outputDir string) error {
 		}
 	}
 
+	// If we find no MCPs present in the directory, then we assume the default
+	// Master, Worker ones and make sure that any PAO manifests that specify a Master or Worker label
+	// get correctly rendered. Specifying MCPs outside of the default will still cause a failure as is expected.
+	if len(mcPools) == 0 {
```

Comment:
There is a corner case here that I am not sure we want address however. An SNO with extra workers might in theory use master + eg. worker-cnf MCPs. The extra worker-cnf will make this condition return false and the master pool will be missing.

Reply:
This is the only concern I have, I would rather create only the pools that are missing.

Eg. if no master is loaded from the folder, inject the master here. The same for worker.

Reply:
Ah my mistake, missed this, updated the code to account for this, please give it another look.